package.json

```diff
@@ -63,8 +44,10 @@
     "eslint-plugin-prettier": "5.1.3",
     "eslint-plugin-require-extensions": "0.1.3",
     "prettier": "3.1.1",
+    "test-exclude": "7.0.1",
```

Comment:
For some reason, `vitest` required this. I'm guessing it's related to the dependency hoisting again.